components/cam/src/physics/crm/rrtmgp/radiation.F90

```diff
@@ -2011,9 +2037,6 @@ subroutine calculate_fluxes_sw(gas_names, gas_vmr, &
       end do
       deallocate(gas_names_lower)
 
-      call handle_error(cld_optics_day%validate())
-      call handle_error(aer_optics_day%validate())
```

Comment:
Were these just redundant?

Reply:
RRTMGP does this internally anyways. This was just an extra check added at some point in the debugging process to catch where things were going awry.